config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users , path: 'show'
```

Comment:
Why add the path as `show`? I don't think this will function correctly. The `root 'user#show'` below should be sufficient.